src/Connection.php

```diff
@@ -54,6 +65,17 @@ public function __construct(array $config)
         $this->useDefaultQueryGrammar();
     }
 
+    /**
+     * Dynamically pass methods to the connection.
+     * @param string $method
+     * @param array $parameters
+     * @return mixed
+     */
+    public function __call($method, $parameters)
+    {
+        return call_user_func_array([$this->db, $method], $parameters);
```

Comment:
The doc block states "pass methods to the connection" but this invokes a method on the `$db` property. Is that a typo or am I misunderstanding something?

Also, I don't think we need `call_user_func_array()` here. See: https://freek.dev/332-making-call-user-func-array-more-readable

Reply:
Done